web/source/resources/osk/kmwosk.css

```diff
@@ -11,10 +11,7 @@
   font-family: SpecialOSK;
   font-style:  normal;
   font-weight: normal;
-  src: url('keymanweb-osk.eot');   
-  src: url('keymanweb-osk.eot?#iefix') format('embedded-opentype'), 
-    url('keymanweb-osk.ttf')  format('truetype'),
-    url('keymanweb-osk.woff') format('woff');     
```

Comment:
We also need to remove corresponding references in android\KMEA\app\src\main\java\com\tavultesoft\kmea\KMManager.java and android\KMEA\build.sh for .eot and .woff.